hooks.yaml

```diff
@@ -0,0 +1,5 @@
+-   id: pydocstyle
```

Comment:
this file is deprecated since v0.12.0 (2017-01-24) and hasn't worked since v0.16.0 (2017-08-01) -- I wouldn't suggest new usages of it

Reply:
I agree, I left it here mainly for the confirmation.